OpenTranslator/Controllers/Awesome/PascalTranslationController.cs

```diff
@@ -226,18 +226,20 @@ enum BlockType
         /// <param name="input"></param>
         /// <param name="LanguageCode"></param>
         /// <param name="filename"></param>
-        void LoadFromReader(TextReader reader, string input, string filename = "")
+        void LoadFromReader(TextReader reader, string input, string filename = "",string Languagecode="")
         {
             //clean error message 
             errorMessage = string.Empty;
             var currentLanguage = _defaultlanguage; //English by default
```

Comment:
just asking: why did you add a param named "LanguageCode" when this method is already using _var currentLanguage_ ?
currentLanguage is set with the stream file language, and if the file don't have language line info, it is set with the default language (english).
In fact, LanguageCode could be an empty string if you don't use it, and that will be an error because every word should belong to a language.

Reply:
I have added language code as parameter because while exporting 
I have called the "LoadFromReader" for all the languages so each time
I have passed the language code so file stream will generate for all languages and get download as zip
Previously it was work only for en language i.e default. So I have added language code parameter.